### PR TITLE
Make mouse more responsive

### PR DESCRIPTION
**Pull request by ad-chaos**
## What kind of change does this PR introduce?
- Fix
supersedes #1511 

## Did this PR introduce a breaking change? 

- No

### CODE REVIEW

**Review by fredizzimo — CHANGES_REQUESTED** [review]
This scrolls 6 lines instead of 3 for each tick on Windows when `mousescroll=ver:3,hor:6` and 2 lines when `mousescroll=ver:1,hor:1`.

It worked perfectly before.

**Comment by ad-chaos** [comment]
I was blocked from discord...

I am not even kidding, I have no idea why and what, I was just typing a response to your comment and suddently kabaam, I get a white screen :((

Till I try figuring that out, Do you agree with the approach taken here? Or do you believe its a winit bug?

**Comment by fredizzimo** [comment]
@ad-chaos. I think discord is partially down or something, my message also got the red color. There's no hurry to figure it out, we can continue when it's up. But we need to find the root cause of the issue.

But floating points are problematic due to drift, on the other platforms as well, so here's what I wrote in discord
>  think at least on Windows we should convert all the values we get from winit to fractions of 120 and do all the calculations as integer. On Wayland and X11, at least the way winit handles it, the input is integers, should the values could multipled by 120 to keep the same logic. 
>
> The macOS implementation is a mystery though, since a single value of 0.1 does not make any sense. A tick should always add up to 1, or close to it allowing floating point errors.

**Comment by ad-chaos** [comment]
I see, I'll read up a bit more on mouse handling till then

File: src/window/mouse_manager.rs [hunk 1]
```diff
@@ -67,6 +67,29 @@ struct TouchTrace {
...
 }
 
+struct ScrolledPixels {
+    x: f32,
+    y: f32,
+}
+
+impl ScrolledPixels {
+    fn new() -> ScrolledPixels {
+        ScrolledPixels { x: 0., y: 0. }
```
Comment (MultisampledNight) [inline src/window/mouse_manager.rs:77]: Could maybe just be `Default::default`.

File: src/window/mouse_manager.rs [hunk 2]
```diff
@@ -67,6 +67,29 @@ struct TouchTrace {
...
+    }
+
+    fn reset(&mut self) {
+        self.x = 0.;
+        self.y = 0.;
```
Comment (MultisampledNight) [inline src/window/mouse_manager.rs:82]:
```suggestion
        *self = Self::new();
```

doesn't require updating another place for theoretical future fields, defaults or type changes.

**Comment by Kethku** [comment]
I'm going to close this for now. @ad-chaos feel free to reopen if you get back to this

**Pull request closed without merging**